lib/config-util.js

```diff
@@ -18,18 +18,45 @@ function guessCWD (cwd) {
   return cwd
 }
 
+function moduleExists (name) {
+  try {
+    return typeof require.resolve(name) !== 'undefined'
+  } catch (e) { return false }
```

Comment:
I missed this before, please add line-feeds around the `return false` and tell nyc to ignore the unreachable code:
```js
try {
  return typeof require.resolve(name) !== 'undefined';
} catch (e) {
  /* istanbul ignore next */
  return false;
}
```